src/main/java/com/epam/processor/DataProcessor.java

```diff
@@ -27,6 +26,12 @@ public DataProcessor(List<RoadAccident> roadAccidentList){
      * @return
      */
     public RoadAccident getAccidentByIndex7(String index){
+        Iterator<RoadAccident> iter = roadAccidentList.iterator();
+        while(iter.hasNext()) {
```

Comment:
Since it's a homework, I'd like to try more way to implement it.
In other function, I use the for(:) style